src/test/java/jenkins/plugins/git/MergeWithGitSCMExtensionTest.java

```diff
@@ -0,0 +1,95 @@
+package jenkins.plugins.git;
+
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+import hudson.plugins.git.*;
+import hudson.plugins.git.extensions.GitSCMExtension;
+import hudson.plugins.git.extensions.GitSCMExtensionTest;
+import hudson.plugins.git.extensions.impl.LocalBranch;
+import hudson.plugins.git.extensions.impl.PreBuildMerge;
+import hudson.plugins.git.util.BuildData;
+import junit.framework.TestCase;
+import nl.jqno.equalsverifier.EqualsVerifier;
+import org.eclipse.jgit.lib.Constants;
+import org.jenkinsci.plugins.gitclient.MergeCommand;
+import org.junit.Test;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
```

Comment:
Having imported all Assert classes with the wildcard import, we don't need to import assertEquals explicitly. I prefer the wildcard import of the Assert classes, so this line can be removed.

The [SCM coding style guide](https://github.com/jenkinsci/scm-api-plugin/blob/master/CONTRIBUTING.md#imports) allows wildcard imports of JUnit and HJamcrest classes.
```suggestion
```